packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -1,24 +1,37 @@
+import { Notice } from '@cerner/terra-docs';
 import { Badge } from '@cerner/terra-functional-testing/package.json?dev-site-package';
 
 <Badge />
 
 # Terra Functional Testing
 
-The terra-functional-testing library is a utility for developing automation tests. The library extends the [WebdriverIO Framework](https://webdriver.io/) to facilitate automating accessibility and functional testing for Terra projects.
+The terra-functional-testing library is a utility for developing automation tests. The library extends [WebdriverIO](https://v6.webdriver.io/) to facilitate automating accessibility and functional testing for Terra projects.
 
-## Getting Started
+## System Requirements
 
-To get started ensure [docker](https://www.docker.com/) has been installed. The terra-functional-testing library uses docker to setup and run tests in a containerized environment.
+- [Node.js](http://nodejs.org/) Install at least v10.24.0. Due to requirements of fibers used within the project install no greater than lts/erbium (v12).
+- [Docker](https://www.docker.com/) Install docker. Docker is used to setup and run tests in a containerized environment.
```

Comment:
I didn't add Mac OS/Linux here as a system requirement, but it might be something we want to consider

Reply:
We could add a caution notice stating that your milage may vary for windows support since we've developed this on mac os/linux

Reply:
I decided not to add anything for now, If this comes up again in the future we can talk through the appropriate messaging and where the messaging should be presented in our doc site